.gitlab-ci.yml

```diff
@@ -133,7 +133,9 @@ run_test_rpm-x64:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/rpm_x64:latest
   tags: [ "runner:main", "size:2xlarge" ]
   script:
-    - inv -e test --race --profile --cpus 4
+    # Exclude systemd because it cannot succeed on Centos 6: the image doesn't have the shared object required by
+    #  https://github.com/coreos/go-systemd/blob/c8cc474ba8655dfbdb0ac7fcc09b7faf5b643caf/sdjournal/functions.go#L46
+    - inv -e test --race --profile --cpus 4 --build-exclude=systemd
```

Comment:
this exclusion is _relatively_ OK as long as we're sure to run these tests on the debian image. Could you add that to the comment?